saleor/graphql/shop/resolvers.py

```diff
@@ -0,0 +1,35 @@
+from ...core.taxes import display_gross_prices
+from ...plugins.manager import get_plugins_manager
+from ...shipping.models import ShippingMethod
+from ..channel import ChannelContext
+
+
+def resolve_available_shipping_methods(channel_slug: str, address):
+    available = ShippingMethod.objects.filter(
+        channel_listings__channel__slug=channel_slug
+    ).price_based()
```

Comment:
I'm not sure if we should return only `price_based` shipping methods or all? I added `price_based` as I know it's for showing minimal order total for free shipping by the storefront. But maybe it will be misleading. @maarcingebala